ethcore/src/evm/interpreter/mod.rs

```diff
@@ -292,14 +292,21 @@ impl<Cost: CostType> Interpreter<Cost> {
 					}
 				};
 			},
-			instructions::CALL | instructions::CALLCODE | instructions::DELEGATECALL => {
+			instructions::CALL | instructions::CALLCODE | instructions::DELEGATECALL | instructions::STATICCALL => {
```

Comment:
It seems that gas calculation for `staticcall` is not implemented (I suppose it should be the same as for `CALL`)